src/main/java/net/earthcomputer/clientcommands/TempRules.java

```diff
@@ -53,6 +54,15 @@ public static void setMaxEnchantItemThrows(int maxEnchantItemThrows) {
     @Rule
     public static boolean infiniteTools = false;
 
+    @Rule
+    public static boolean chorusManipulation = false;
+
+    @Rule(readOnly = true)
+    //If the goal is relative to the player
+    public static boolean chorusRelativeTel;
+    public static Vec3d chorusGoalV1;
+    public static Vec3d chorusGoalV2;
```

Comment:
Don't put additional variables in `TempRules`. If you can't think of anywhere to put it, you can make a new class inside the `features` package.